ui-components/src/prototypes/Card/CardContainer.stories.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react"
```

Comment:
My understanding of the prototype folder was that it is for components that were in a beta / unreleased but with the intention to be released eventually - what's the goal w these prototypes here?

Reply:
I can remove it, but I'm hoping first it can spark some feedback and discussion (as I mentioned in the longer comment above)…